### ISSUE TITLE
Add windows host, does not support the specified public IP

### ISSUE THREAD

**Issue by kingsd041** [rancher/rancher#10590]
Rancher versions:
rancher/server: 1.6.13-rc3
rancher/agent:

Docker version: (docker version,docker info preferred)
17.06.2-ee-6
Operating system and kernel: (cat /etc/os-release, uname -r preferred)
Windows Server 2016 Standard
Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)
VirtualBox
Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)
single node rancher
Environment Template: (Cattle/Kubernetes/Swarm/Mesos)
**Windows**
**Steps to Reproduce:**
1. Add Host, Step 5 Set the host public IP to 10.0.2.15.
2. Execute the shell from step 7 in PowerShell.
3. Wait for the host to add success.
4. View the host IP in INFRASTRUCTURE
**Results:**
The host address is 192.168.99.104, not 10.0.2.15 specified in step 5

My host network settings：
```
PS C:\Users\Administrator> ipconfig /all



以太网适配器 以太网:

   连接特定的 DNS 后缀 . . . . . . . : lan
   描述. . . . . . . . . . . . . . . : Intel(R) PRO/1000 MT Desktop Adapter
   物理地址. . . . . . . . . . . . . : 08-00-27-93-34-BD
   DHCP 已启用 . . . . . . . . . . . : 是
   自动配置已启用. . . . . . . . . . : 是
   本地链接 IPv6 地址. . . . . . . . : fe80::3597:cb1a:accd:f218%11(首选)
   IPv4 地址 . . . . . . . . . . . . : 10.0.2.15(首选)
   子网掩码  . . . . . . . . . . . . : 255.255.255.0
   获得租约的时间  . . . . . . . . . : 2017年12月15日 17:50:03
   租约过期的时间  . . . . . . . . . : 2017年12月16日 17:50:02
   默认网关. . . . . . . . . . . . . : 10.0.2.2
   DHCP 服务器 . . . . . . . . . . . : 10.0.2.2
   DHCPv6 IAID . . . . . . . . . . . : 34078759
   DHCPv6 客户端 DUID  . . . . . . . : 00-01-00-01-21-C0-4F-94-08-00-27-93-34-BD
   DNS 服务器  . . . . . . . . . . . : 192.168.2.1
   TCPIP 上的 NetBIOS  . . . . . . . : 已启用

以太网适配器 以太网 2:

   连接特定的 DNS 后缀 . . . . . . . :
   描述. . . . . . . . . . . . . . . : Intel(R) PRO/1000 MT Desktop Adapter #2
   物理地址. . . . . . . . . . . . . : 08-00-27-D1-EA-44
   DHCP 已启用 . . . . . . . . . . . : 是
   自动配置已启用. . . . . . . . . . : 是
   本地链接 IPv6 地址. . . . . . . . : fe80::38ec:e5d3:ffe4:a849%16(首选)
   IPv4 地址 . . . . . . . . . . . . : 192.168.99.104(首选)
   子网掩码  . . . . . . . . . . . . : 255.255.255.0
   获得租约的时间  . . . . . . . . . : 2017年12月15日 17:50:03
   租约过期的时间  . . . . . . . . . : 2017年12月15日 18:40:03
   默认网关. . . . . . . . . . . . . :
   DHCP 服务器 . . . . . . . . . . . : 192.168.99.6
   DHCPv6 IAID . . . . . . . . . . . : 302514215
   DHCPv6 客户端 DUID  . . . . . . . : 00-01-00-01-21-C0-4F-94-08-00-27-93-34-BD
   DNS 服务器  . . . . . . . . . . . : fec0:0:0:ffff::1%1
                                       fec0:0:0:ffff::2%1
                                       fec0:0:0:ffff::3%1
   TCPIP 上的 NetBIOS  . . . . . . . : 已启用

以太网适配器 以太网 3:

   连接特定的 DNS 后缀 . . . . . . . :
   描述. . . . . . . . . . . . . . . : Intel(R) PRO/1000 MT Desktop Adapter #3
   物理地址. . . . . . . . . . . . . : 08-00-27-F9-F8-7D
   DHCP 已启用 . . . . . . . . . . . : 否
   自动配置已启用. . . . . . . . . . : 是
   本地链接 IPv6 地址. . . . . . . . : fe80::5cd7:b7d8:3fb3:ca07%12(首选)
   IPv4 地址 . . . . . . . . . . . . : 172.22.101.200(首选)
   子网掩码  . . . . . . . . . . . . : 255.255.255.0
   默认网关. . . . . . . . . . . . . :
   DHCPv6 IAID . . . . . . . . . . . : 503840807
   DHCPv6 客户端 DUID  . . . . . . . : 00-01-00-01-21-C0-4F-94-08-00-27-93-34-BD
   DNS 服务器  . . . . . . . . . . . : fec0:0:0:ffff::1%1
                                       fec0:0:0:ffff::2%1
                                       fec0:0:0:ffff::3%1
   TCPIP 上的 NetBIOS  . . . . . . . : 已启用

```

**Comment by deniseschannon** [comment]
Available with v1.6.13-rc4

**Comment by kingsd041** [comment]
Tested with Rancher/server:v1.6.13-rc4.
Test passed.
